web/default/style.css

```diff
@@ -265,13 +265,14 @@ label {
 }
 
 #Masthead {
+    font-family: monospace; /* ala <kbd> */
```

Comment:
It's not quite visible on the screenshot however I'd add a margin/padding for the bottom part of the info text. There is a little space at the top of it and so there should be a little space under it. I mean it's almost touching the orange line.

![screenshot from 2017-01-18 14-33-23](https://cloud.githubusercontent.com/assets/6997160/22066008/1f8313e4-dd8b-11e6-9e31-8fad370e664d.png)